<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewClaims</class>
 <widget class="QWidget" name="viewClaims">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pay">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Claim as  paid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Claim ID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
